.github/workflows/CI_matlab.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         -   name: Code coverage
             uses: codecov/codecov-action@v5
             with:
-                file: coverage.xml
+                files: coverage.xml
```

Comment:
unrelated but was flagged in my IDE as not compatible with the codecov-action schema